### ISSUE TITLE
Apple loggin user's name is shown as email in the app

### ISSUE THREAD

**Issue by hsb1007** [kuky-com/mobile-app#46]

**Comment by tungmt** [comment]
It is apple quite stupid feature. They only provide user full name first time they open the app. Then in the review they not allow to edit or add name on the onboarding step. So for now the only way is allow customer edit name in profile screen.

**Comment by hsb1007** [comment]
Yes during the onboarding, we should save their detail into our database and profile photos if exists

**Comment by hsb1007** [comment]
![Image](https://github.com/user-attachments/assets/3e8b2a5b-8df6-4e16-ba71-512434f0a2d7)

I can see this is still happening. 

If apple only provide first time they open the app, we should store those information in our db so that we can use it.


This user detail seems spammy / some kinds of system testing

**Comment by tungmt** [comment]
It already solved!
